ui/raidboss/emulator/data/NetworkLogConverter.ts

```diff
@@ -0,0 +1,41 @@
+import EventBus from '../EventBus';
+import LineEvent from './network_log_converter/LineEvent';
+import LogRepository from './network_log_converter/LogRepository';
+import ParseLine from './network_log_converter/ParseLine';
+
+export default class NetworkLogConverter extends EventBus {
+  convertFile(data: string): LineEvent[] {
+    const repo = new LogRepository();
+    return this.convertLines(
+        // Split data into an array of separate lines, removing any blank lines.
+        data.split(NetworkLogConverter.lineSplitRegex).filter((l) => l !== ''),
+        repo,
+    );
+  }
+
+  convertLines(lines: (string | LineEvent)[], repo: LogRepository): LineEvent[] {
```

Comment:
Who is calling `convertLines` with `LineEvent[]`?

Reply:
This has to do with the conversation below related to memory. Depending on how we solve that, this will likely change.